tracer/tracer.go

```diff
@@ -176,7 +176,8 @@ func (t *Tracer) NewRootSpan(name, service, resource string) *Span {
 	span := NewSpan(name, service, resource, spanID, spanID, 0, t)
 
 	span.buffer = newSpanBuffer(t.channels, 0, 0)
-	t.sampler.Sample(span)
+	t.Sample(span)
+	// [TODO:christian] introduce distributed sampling here
```

Comment:
curious about this TODO. is it going to be merged in master and done in another PR?
i just want to make sure it's not going to stay in the code for too long

Reply:
OK, the TODO here means (I'm going to explain it in the comment I think): currently, the lib does not set the sampling priority on new root spans. This means there's no way a trace coming from this library (the top root comes from it) could have `_sampling_priority_v1` defined. So when we decide to do this, this is the place where we should call something like `span.SetSamplingPriority(aValueWeGetFromTraceAgentFeedback)`